### ISSUE TITLE
Macbook Pro / Can't go into fullscreen mode

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#251]
Originally reported on Google Code with ID 252

```
Help me please, I don't know, what to do..

I use Mupen+ (on a Macbook Pro) with the Rice Video Plugin 1.5 and when I try to get
into the 
fullscreen mode, the screen, where the game runs, turns black and then it goes back
to the game 
but it's still a window game, so my fullscreen doesn't work.. Can anybody help me?


Sorry for my bad english, I don't speak it very much.. ^^
```

Reported by `henkesdominique` on 2009-08-07 07:44:08

**Comment by richard42** [comment]
```
The last time I tried fullscreen on Macs was only working with the Glide64 plugin.
```

Reported by `preotgeerrs` on 2009-08-07 13:49:45

**Comment by richard42** [comment]
```
Go to configure mupen64plus click configure under the options key and click the
hotkeys tab.  Once under the hotkeys tab map out a button on your controller to
toggle between fullscreen. Also make sure the resolution is set to 640x480 in
fullscreen in graphic settings. Using this method is the only way I could get it to
go fullscreen. I hope this helps

```

Reported by `JPtrickMay` on 2009-11-04 17:58:18

**Comment by richard42** [comment]
```
This was probably the same issue as reported in #249, which has been fixed in the
latest code.  Please test with the 1.99.1 beta release and re-open this bug report
if
the problem remains.
```

Reported by `richard42g` on 2010-01-12 22:26:05
- **Status changed**: `Duplicate`
- **Merged into**: #249
